include/NovelRT/Input/InteractionObject.h

```diff
@@ -0,0 +1,26 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_INPUT_INTERACTIONOBJECT_H
+#define NOVELRT_INPUT_INTERACTIONOBJECT_H
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
+
+namespace NovelRT::Input {
+  class InteractionObject : public WorldObject {
+    friend class InteractionService; //how to make shit tightly coupled oh god
```

Comment:
we should log bugs for these comments or consistently prefix with `TODO` (or both) 😄

Reply:
Do we even still need this with the centralised header?